README.md

```diff
@@ -90,6 +90,23 @@ spill foo
 // console.log(foo)
 ```
 
+#### Comment
+```
+IMO ini tuh ga penting
+// // ini tuh ga penting
+```
+
+#### Increment/Decrement
+```
+literally circleku itu 0
+// let circleku = 0
+
+circleku toxic
+// circleku--
+
+circleku healthy
+// circleku++
+```
```

Comment:
I like it, but i'm afraid this will collide to other function. my suggest to change this to `circleku-toxic` and `circleku-healthy`

Reply:
or using `toxic circle` and `healthy circle`, `ku` in here make the words larger.

Reply:
I add 'dash' just exclusive for this function. but I agree to you to make this function similar format to other

Reply:
I'm updating the example to `circle toxic` and `circle healthy`, I believe they still have the same meaning
I can't use `toxic circle` because another parser is already using it

As for the `toxic` conflict issue, seems `.* toxic` and `toxic .*` can still be differentiated by the regexes